de.metas.handlingunits.base/src/main/java/de/metas/handlingunits/inventory/impl/HUInternalUseInventoryProducer.java

```diff
@@ -103,14 +105,15 @@ private HUInternalUseInventoryProducer()
 		return result;
 	}
 
-	private final List<I_M_Inventory> createInventories(final int warehouseId,
+	private final List<I_M_Inventory> createInventories(
+			final WarehouseId warehouseId,
 			final List<I_M_HU> hus,
 			final ActivityId activityId,
 			final String description,
 			final boolean isCompleteInventory,
 			final boolean isCreateMovement)
 	{
-		final I_M_Warehouse warehouse = InterfaceWrapperHelper.loadOutOfTrx(warehouseId, I_M_Warehouse.class);
+		final I_M_Warehouse warehouse = Services.get(IWarehouseDAO.class).getById(warehouseId);
```

Comment:
you can basically still use `(InterfaceWrapperHelper.)loadOutOfTrx(warehouseId, I_M_Warehouse.class);` as before, with the RepoIdAware as parameter